addons/payment/models/payment_transaction.py

```diff
@@ -781,19 +782,20 @@ def _execute_callback(self):
 
             valid_callback_hash = self._generate_callback_hash(model_sudo.id, res_id, method)
             if not consteq(ustr(valid_callback_hash), callback_hash):
-                _logger.warning("invalid callback signature for transaction with id %s", tx.id)
+                _logger.warning("invalid callback signature for transaction with reference %s",
+                                tx.reference)
```

Comment:
In our scope, we prefer wrapping long lines as follows when the function's parameters don't all fit on one line:
```suggestion
                _logger.warning(
                    "invalid callback signature for transaction with reference %s",
                    tx.reference
                )
```
However, they do fit in this case so you can simply do this:
```suggestion
                _logger.warning(
                    "invalid callback signature for transaction with reference %s", tx.reference
                )
```